embrace-gradle-plugin/src/test/java/io/embrace/android/gradle/plugin/ndk/NdkUploadTasksRegistrationTest.kt

```diff
@@ -187,11 +187,10 @@ class NdkUploadTasksRegistrationTest {
         assertTaskRegistered(EncodeFileToBase64Task.NAME, testAndroidCompactedVariantData.name)
     }
 
-    // TODO: Do all Unity projects have a mergeNativeLibs task?
     @Test
     fun `an error is thrown when merge native libs task is not found`() {
         // Given a project where merge native libs task is not registered
-        val projectTypeProvider = project.provider { ProjectType.UNITY }
+        val projectTypeProvider = project.provider { ProjectType.NATIVE }
```

Comment:
Is this change necessary?

Reply:
oh not at all, I thought I removed that change sorry